scale/recipe/models.py

```diff
@@ -1843,43 +1272,18 @@ def edit_recipe_type_v6(self, recipe_type_id, title, description, definition, au
                 msgs = [create_sub_update_recipe_definition_message(id, recipe_type.id) for id in super_ids]
                 CommandMessageManager().send_messages(msgs)
 
-    def get_by_natural_key(self, name, version):
+    def get_by_natural_key(self, name, revision_num):
         """Django method to retrieve a recipe type for the given natural key
 
         :param name: The human-readable name of the recipe type
         :type name: string
+        :param revision_num: The revision number of the recipe type
+        :type revision_num: int
```

Comment:
See earlier comment about version != revision_num

Reply:
Removed argument as recipe_types don't require anything other than name to be unique